### PR TITLE
Changed the Twitter logo

### PR DESCRIPTION
**Pull request by MayaSatishRao**
Resolve https://github.com/opencollective/opencollective/issues/7195
# Description
New Twitter (X) logo is added in the footer and header of open collective page

# Screenshots
Screenshot 2024-01-08 200030
Screenshot 2024-01-08 200109

### CODE REVIEW

File: components/collectives/sections/LearnMore.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const learningChannels = [
   {
     id: 'openCompany',
     name: 'Open Company',
-    link: 'https://drive.opencollective.com',
+    link: 'https://opencollective.com/opencollective',
```
Comment (Betree) [inline components/collectives/sections/LearnMore.js:98]: This seems unrelated to the current PR, can you maybe do an interactive rebase?

File: components/ContributorsGrid.js
Comment (Betree) [inline components/ContributorsGrid.js:1]: This seems unrelated to the current PR (updating the Twitter logo). Can you maybe do an interactive rebase?

File: components/ContributorsGrid.js [hunk 2]
```diff
@@ -221,6 +223,9 @@ ContributorsGrid.propTypes = {
 
   /** Collective id */
   collectiveId: PropTypes.number,
+
+  /* Reference to the FixedSizedGrid */
+  gridRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({ current: PropTypes.any })]),
```
Comment (Betree) [inline components/ContributorsGrid.js:228]: This seems unrelated to the current PR (updating the Twitter logo). Can you maybe do an interactive rebase?

Comment (MayaSatishRao) [reply components/ContributorsGrid.js]: @Betree I have made the changes

File: components/ContributorsGrid.js [hunk 3]
```diff
@@ -138,7 +138,6 @@ const ContributorsGrid = ({
 }) => {
   const maxNbRows = maxNbRowsForViewports[viewport];
   const [nbCols, nbRows] = getItemsRepartition(contributors.length, width, maxNbRows);
-
```
[the remark this answers is not in the document]
Comment (MayaSatishRao) [reply components/ContributorsGrid.js]: @Betree I have made the changes. And thanks for the resource that you shared.

**Comment by Betree** [comment]
I haven't tried locally, but this crashes the homepage on Vercel: https://opencollective-frontend-git-fork-mayasati-72655d-opencollective.vercel.app.

```
[Error: EMFILE: too many open files, open '/var/task/node_modules/@icons-pack/react-simple-icons/icons/SiGooglemaps.mjs'] {
  errno: -24,
  code: 'EMFILE',
  syscall: 'open',
  path: '/var/task/node_modules/@icons-pack/react-simple-icons/icons/SiGooglemaps.mjs',
  page: '/home'
}
Error: Runtime exited with error: exit status 1
Runtime.ExitError
```

**Comment by Betree** [comment]
> @Betree I found this [phosphor-icons/react#45 (comment)](https://github.com/phosphor-icons/react/issues/45#issuecomment-1760724607) and tried to make the changes accordingly. It was working fine locally but can you check if this resolves the error

Still getting the same issue

image

**Comment by MayaSatishRao** [comment]
@betree what should I do to remove this error? Should I use a different library instead of react-simple-icons?

**Comment by Betree** [comment]
> @betree what should I do to remove this error?

I have no clue on how to resolve this, and the resolution of this issue is not a priority so I can't invest time in researching solutions.

If you don't have the time to research it yourself, I suggest closing the PR and moving on to something more straightforward.

**Comment by Betree** [comment]
I don't have the bandwidth to research and decide on an icon library. I suggest closing the PR if we can't make it work with the initial specs.

**Comment by MayaSatishRao** [comment]
@Betree I will look again for a solution. Please do not close this issue till then.

**Comment by MayaSatishRao** [comment]
@Betree Can you check it now?

**Review by Betree — APPROVED** [review]
This seems to work; thanks for iterating @MayaSatishRao!

The last thing we must do is rebase the PR on `main` to solve the conflicts.

**Comment by MayaSatishRao** [comment]
@Betree Do I need to perform rebase to solve conflict?

**Comment by Betree** [comment]
If you're not comfortable with it, I can take care of that part

**Review by Betree — COMMENTED** [review]
@MayaSatishRao Your fix doesn't seem to work with the latest version of @icons-pack/react-simple-icons

**Comment by MayaSatishRao** [comment]
I updated react-simple-icons package to latest version and checked. It is working fine locally

**Comment by Betree** [comment]
Resolved in https://github.com/opencollective/opencollective-frontend/pull/10810
